lesson02/assignment/index.html

```diff
@@ -1,14 +1,37 @@
+<!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
-        <title>Assignment 02</title>
-    </head>
-    <body>
-        <main class="main">
-            <h1>title</h1>
-        </main>
-    </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="css/style.css" />
+  <script src="https://kit.fontawesome.com/f399a12497.js" crossorigin="anonymous"></script>
+  <title>Employee Card</title>
+</head>
+
+<body>
+  <main class="main">
+  <div class="card-container">
+    <!--begin div for employee card-->
+    <div class="img">
+      <img src="./img/unsplash-headshot.jpg" alt="Headshot of Steve Smith"></img>
+      <h1>Steve Smith</h1>
+      <h2><span class="italic">Project Manager</span></h2>
+    </div>
+
+    <div class="profile">
+      <p><span class="bold">Company:</span> Front End Dev Co</p>
+      <p><span class="bold">Experience:</span> 3 years</p>
+      <p><span class="bold">School:</span> UW</p>
+      <p><span class="bold">Major:</span> Marketing</p>
+      <p><span class="bold">Email:</span> steve@fedc.com</p>
+      <p><i class="fab fa-linkedin"></i>  <span class="linked-in">steve.linkedinprofile.com</span></p>
+    </div>
+  </div>
+  <!--end div for employee card-->
```

Comment:
Are there ways to better leverage semantic HTML elements, rather than use so many `<div>`s? Maybe the div opened on line 15 would make sense as a `<section>`? Instead of a div wrapped around the image with headlines inside, maybe a [`<figure>` and `<figcaption>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption) could work?